docs/articles/documentation/examples/README.md

```diff
@@ -30,6 +39,10 @@ Example | Description
 [Extract Code to Helpers](https://github.com/DevExpress/testcafe-examples/tree/master/examples/extract-code-to-helpers) | Extract test code to functions defined in a separate module.
 [Create Data-Driven Tests](https://github.com/DevExpress/testcafe-examples/tree/master/examples/create-data-driven-tests) | Create a test that performs parameterized actions.
 [Use Page Model](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-page-model) | Extract element selectors and common operations with these elements to a page model.
+[Pass Parameters to Tests](https://github.com/DevExpress/testcafe-examples/tree/master/examples/pass-parameters-to-tests) | Pass parameters to TestCafe tests.
```

Comment:
The `Extended Client-Side Error Tracking`, `Set a Custom Referrer`, `Check the Downloaded File Name and Content` examples are not related to `Test Organization` category.

I propose to create 2 new categories and put them there.

**Upload/Download**
[Check the Downloaded File Name and Content]
[Select Files To Upload]

**Control HTTP requests**
[Set a Custom Referrer]

The [Extended Client-Side Error Tracking] put to the 'Interoperability' category.

Reply:
edited